cluster/images/hyperkube/Makefile

```diff
@@ -84,6 +84,8 @@ else
 endif
 	# Download CNI
 	curl -sSL --retry 5 https://storage.googleapis.com/kubernetes-release/network-plugins/cni-${ARCH}-${CNI_RELEASE}.tar.gz | tar -xz -C ${TEMP_DIR}/cni-bin
+	curl -sSL --retry 5 -o  ${TEMP_DIR}/cni-bin/bin/calico https://github.com/projectcalico/calico-cni/releases/download/v1.9.1/calico
```

Comment:
there is v1.11.0 out https://github.com/projectcalico/calico-cni/releases.

please ignore me if this is not relevant.